src/main/java/chopchop/logic/LogicManager.java

```diff
@@ -0,0 +1,112 @@
+package chopchop.logic;
```

Comment:
wait a minute, isn't this file a duplicate of the `CommandDispatcher`

Reply:
Deleting this old and deproved version.